x/msgfees/types/msgs.go

```diff
@@ -50,9 +55,29 @@ func (msg MsgAssessCustomMsgFeeRequest) ValidateBasic() error {
 	if !msg.Amount.IsPositive() {
 		return errors.New("amount must be greater than zero")
 	}
+	_, err = msg.GetBips()
+	if err != nil {
+		return err
+	}
 	return nil
 }
 
+// GetBips converts the msg RecipientBasisPoints to a uint32 basis point value 0 - 10,000
```

Comment:
This goes in combination with the comment I made on line `#72`. Not certain if there's a need to mention the value ranger in the comments. If so, it should be something like  `0 - RECIPIENT_MAX_BIPS` or something like that.

Reply:
Along the lines as above.  This also makes it more readable without the user having to refer to what a constant is set at.

Reply:
👍